static/js/settings_org.js

```diff
@@ -673,7 +673,7 @@ exports.build_page = function () {
         });
 
         var save_btn_controls = subsection.find('.subsection-header .save-button-controls');
-        var button_state = show_change_process_button ? "unsaved" : "saved";
+        var button_state = show_change_process_button ? "unsaved" : "discarded";
```

Comment:
Can you explain what precisely this change does?  I don't understand what `show_change_process_button` means in this code path.

Reply:
I've explained this in a commit message,
> In "discarded" state, "Save/discard" widgets start disappearing without the
lag of 800ms as in "saved" state. Furthermore, when all settings are restored to
the original state (i.e. there is no net change in the state of input elements),
"discard" state is more intuitive.

And if you are asking about why we have this at all, then it is to `hide` and `show` the widgets.